### DISCUSSION TITLE
Play 2.8 support

### DISCUSSION THREAD

**Discussion by felipebonezi** [open-telemetry/opentelemetry-java-instrumentation#6150] [Q&A] [answered]
Hi everyone, 🤝 

As we can see on [Supported Libraries & Frameworks](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md) Play it isn't included as supported yet.

Do you have any plans to support this version?

**Comment by laurit** [comment, marked as answer]
That info seems to be outdate as tests actually pass with latest play 2.8

**Reply by felipebonezi** [reply]
Thanks for your PR to confirm.

Ref #6157
